edflow/hooks/logging_hooks/pt_profiling.py

```diff
@@ -0,0 +1,59 @@
+import os
+import torch
+from datetime import datetime
+
+from edflow.hooks.hook import Hook
+from edflow.project_manager import ProjectManager as P
+from edflow.custom_logging import get_logger
+
+
+class PyProfilingHook(Hook):
+    '''Allows to profile your pytorch code!
+
+    Best of all: It allows you to create chrome traces, which can be view
```

Comment:
view -> viewed

Reply:
btw. your docstring is great and allows one to get going immediately, but if you have a link to an exhaustive documentation of the profiler you could add it, too!